### ISSUE TITLE
Cannot Jump to the target page when click the slide's link 

### ISSUE THREAD

**Issue by royalbluewa3** [Molunerfinn/hexo-theme-melody#49]
## I want to create a new issue  

- [x] Yes, I have read [Hexo Docs page](https://hexo.io/docs/), especially [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I have read [Hexo-theme-melody Documentation](https://molunerfinn.com/hexo-theme-melody-doc/).
- [x] And yes, I already searched for current [issues](https://github.com/Molunerfinn/hexo-theme-melody/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

## Melody Information

**Melody Version: 1.5.0**

**Platform: win7 x64**

**Browser: Chrome 65.0**

## Expected behavior 
Jump to the target page when click the link in homepage slides' overview.
## Actual behavior 

![snipaste_20180318_184215](https://user-images.githubusercontent.com/17197502/37565420-24dddb1a-2ae4-11e8-9030-fbb384beea92.png)

## Steps to reproduce the behavior 
--
## Feature Request

---

**Comment by Molunerfinn** [comment]
Maybe it's the PWA cache. You can use `ctrl+f5` multi times & wait the first time to load the reveal.js.
The first time will take some time. Just wait for a while.
![image](https://user-images.githubusercontent.com/12621342/37566249-910072aa-2af1-11e8-93c1-85770d2665a2.png)

**Comment by royalbluewa3** [comment]
Actually...I mean condition like this: display nothing after I click the link config~
![snipaste_20180318_212130](https://user-images.githubusercontent.com/17197502/37566344-af589a06-2af2-11e8-984e-d19354fd96d2.png)

**Comment by Molunerfinn** [comment]
@royalbluewa3 Because the slides in the home page is in a `iframe` element. So the `a` link will be blocked by the browser with the security reason. That's not a bug but may be not good for users. You can click the `config` link in the detail page: https://molunerfinn.com/slide-support/

**Comment by royalbluewa3** [comment]
Got it, thanks~

**Comment by Molunerfinn** [comment]
Haha, you are welcome~ :)
